### DISCUSSION TITLE
Contributing upstream documentation for deploying Valkey

### DISCUSSION THREAD

**Discussion by colinmixonn** [valkey-io/valkey#708] [Q&A]
Hello, I am working with a team at Microsoft/Azure that documents how to deploy Valkey on Azure Kubernetes Service. We want to contribute this documentation upstream to https://valkey.io/docs/. Currently, we are unsure how to contribute and if this is the right place. Is anyone able to provide details about the doc contributing process? Thanks for your help :).

**Comment by stockholmux** [comment]
hey @colinmixonn! I think you want to look at [valkey-doc](https://github.com/valkey-io/valkey-doc), specifically the `/topics/` folder. 

Those markdown files get merged at build time into the [website](https://github.com/valkey-io/valkey-io.github.io) and (separately) generate things like man pages.

**Reply by colinmixonn** [reply]
Thank you @stockholmux! Will try that now that we are ready :)
